Source/VMware.vSphereDSC/Tests/Integration/VMHostVss/VMHostVss.Integration.Tests.ps1

```diff
@@ -45,7 +53,6 @@ $script:configWithRemoveVss = "$($script:dscResourceName)_Remove_Config"
 $script:vmHost = $null
```

Comment:
This variable here is not used so you can remove it.

Reply:
Changed